docs/fast_data/runtime_management.mdx

```diff
@@ -0,0 +1,148 @@
+---
+id: runtime_management
+title: Runtime Management
+sidebar_label: Runtime Management
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+
+# Runtime Management
+
+Welcome to Runtime Management - your tool for achieving better control over the Fast Data environment during runtime.
+This platform empowers you to dynamically manage the execution of Fast Data, offering functionalities to pause and resume microservices effortlessly.
```

Comment:
I would not use this term, since it's quite misleading

Reply:
```suggestion
This component empowers you to dynamically manage the execution of Fast Data, offering functionalities to pause and resume microservices effortlessly.
```